Bottom: b96e947edf1ce9762ed303ce63dbffa641c987a0
Top:    175144b8bfcf0a0742d6ac540d828e8c42a5ab50
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-08 21:29:21 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
